documentation/source/Policies/storage_policy.rst

```diff
@@ -0,0 +1,127 @@
+
+Storage Policies
+==================
+You can now define policies based on the type of application at the storage-class level. Following are some of the properties that can be customized at the default level in the openebs-storageclasses.yaml file.
+
+* jiva-replica-count
+* jiva-replica-image
+* jiva-controller-image
+* volume capacity and monitoring 
+
+Replica Count Policy
+----------------------
+You can specify the number of jiva replicas you want to create. In the following example, the jiva-replica-count is specified as 1. Hence, a single replica is created.  
+::
+
+    # Define a storage classes supported by OpenEBS
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: openebs-standalone
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+        pool: hostdir-var
+        openebs.io/jiva-replica-count: "1"
+    size: 5G
+
+Replica Image Policy
+----------------------
+You can add a replica image using the jiva-replica-image property. Specify the value in the "repo-hub-name/project-name:tag-name" format. In the following example, openebs is the repository name and jiva:0.4.0 is the project and tag name.
+::
+
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: mysql
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+        openebs.io/jiva-replica-count: "1"
+        openebs.io/capacity: "1G"
+        openebs.io/jiva-replica-image: "openebs/jiva:0.4.0"
+  
+Controller Image Policy
+---------------------------
+You can add a jiva controller image using the jiva-controller-image property. Specify the value in the ""repo-hub-name/project-name:tag-name" " format. In the following example, openebs is the repository name and jiva:0.4.0 is the project and tag name.
```

Comment:
No need to talk about formating. This is a well known concept.

The final wordings can be:
`You can specify the jiva controller image using openebs.io/jiva-controller-image property. 
NOTE: jiva controller image is a docker image.`

Reply:
Done